packages/relay/src/TransactionManager.ts

```diff
@@ -175,7 +207,14 @@ data                     | ${transaction.data}
       })
       this.logger.debug(`sendTransaction: gasLimit from estimate: ${gasLimit}`)
     }
-    await this.validateBalance(txDetails.signer, maxFeePerGas, gasLimit)
+    const signerBalance = await this.contractInteractor.getBalance(txDetails.signer)
+    const {
+      requiredBalance,
+      isSufficient
+    } = await this.validateBalance(txDetails.signer, maxFeePerGas, gasLimit, signerBalance)
+    if (!isSufficient) {
```

Comment:
Isn't that a duplicate since you already check it inside `validateBalance()`?

Reply:
No. This is the `sendTransaction` method and it throws on insufficient balance.
The `validateBalance` does not throw any more - I don't want to throw in the `resendTransaction` as this is just the boosting - I want to return.